classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] Kindly ensure that the 'has-many' side (a classroom has many students) is being set up so that a newly created student can be added to a classroom. See suggestion below:

```suggestion
  def add_student(student)
    @students.push(student)
    student.classroom = self
  end
```